route53_swatt_app.tf

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# Resource records that support SWATT
```

Comment:
You have to at least define what SWATT stands for.  I've never heard of it before, so I effectively don't know what this DNS record is for.

Reply:
See [18c9f13](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/88/commits/18c9f13ba2fe6c83b99b5dbcb26ddd8dcc323cab)